Formula/arduino-cli.rb

```diff
@@ -17,15 +17,18 @@ class ArduinoCli < Formula
     sha256 cellar: :any_skip_relocation, big_sur:       "698b89996bbb1a67f26e4344fdd1b5f8993f59f14c8cb774309938ed4fac4ebc"
     sha256 cellar: :any_skip_relocation, catalina:      "2980a7274dccb3ca12c2ca7a54696a2cf8fd086a93edebdee363f75f7c0e1ffa"
     sha256 cellar: :any_skip_relocation, mojave:        "5cea6da20a71ee40a617e54fa52a0e64951acdc4bbf216bbd7bbc954fee924b1"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "8a4885ca14f11dc6eefaf7238f81d6024850e4d093b2893afad21337a1685847"
   end
 
   depends_on "go" => :build
 
   def install
+    time = `date -u +"%Y-%m-%dT%H:%M:%SZ"`.chomp
     ldflags = %W[
       -s -w
       -X github.com/arduino/arduino-cli/version.versionString=#{version}
-      -X github.com/arduino/arduino-cli/version.commit=#{Utils.git_head}
+      -X github.com/arduino/arduino-cli/version.commit=#{Utils.git_head(length: 8)}
+      -X github.com/arduino/arduino-cli/version.date=#{time}
```

Comment:
```suggestion
      -X github.com/arduino/arduino-cli/version.date=#{Time.now.utc.iso8601}
```
Let's follow other formulae here for passing the current time to the build. This should be equivalent to calling `date -u +"%Y-%m-%dT%H:%M:%SZ"` in the shell.